src/main.rs

```diff
@@ -412,6 +435,12 @@ extern "C" fn kernel_init(hart_id: u64, fdt_addr: u64) {
     SbiConsole::set_as_console();
     println!("Salus: Boot test VM");
 
+    // No return value. Just leave a message and hang
```

Comment:
Super nit: Just reduce to "Will panic if register width > 256 bits"?

Reply:
fixed